src/autocapture-utils.ts

```diff
@@ -338,31 +338,21 @@ export function getDirectAndNestedSpanText(target: Element): string {
  */
 export function getNestedSpanText(target: Element): string {
     let text = ''
-    if (target && target.nodeType === 1 && target.children?.length > 0) {
-        for (const child of target.children) {
-            if (child && child.nodeType === 1 && child.tagName?.toLowerCase() === 'span') {
+    if (target && target.childNodes && target.childNodes.length) {
```

Comment:
It doesn't actually make a tonne of sense to use `childNodes` since that will include comment and text nodes as well as elements. But none of this makes sense and we don't get (or don't notice) errors from `getSafeText` so I wanted to shift this to be as similar as possible.

It iterates `childNodes` with `_each` so that's what I've changed this to do

We check `target.childNodes` && `target.childNodes.length` so it is implicit that `Element` is not a text or comment node. We then check that the child has a `tagName === span`, so it is implicit there too